numpy/core/src/umath/loops.c.src

```diff
@@ -786,6 +795,14 @@ NPY_NO_EXPORT void
     }
 }
 
+NPY_NO_EXPORT void
+@TYPE@_isfinite(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func))
+{
+    OUTPUT_LOOP {
+        *((@type@ *)op1) = 1;
```

Comment:
I am wondering whether we should do this, implement `np.isnat` instead, or do both (and if we implement isnat, do I have to do this user type resolution correctly like you did.